docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx

```diff
@@ -0,0 +1,56 @@
+---
+keywords: [intermediate, governance, tutorial, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import '/src/components/CenterImages/center.scss';
+
+# Accessing Neurons Created with Quill using the NNS Dapp and Ledger
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Tutorial"]} />
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/lKbYUmqF7J4?si=jIvx5rEza7BnMchE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## Overview
+You may have neurons controlled by a principal ID that was created using [quill](../../../../../../docs/current/developer-docs/developer-tools/cli-tools/quill-cli-reference/quill-parent). In this case, it may be inconvenient to manage your neurons through the terminal. If you know the seed phrase of this principal ID, you can use a Ledger hardware wallet to manage your neurons through the [NNS dapp](/https://nns.ic0.app/).
+
+## How to access neurons controlled by a principal ID created with quill
+
+- **Step 1**: Unlock your Ledger device and navigate to **Settings Manage device** > **Security** > **Reset device**
+
+![NNS Dapp Ledger 1](../../_attachments/ledger-1.webp)
+
+- **Step 2**: Type in your PIN to reset your Ledger device.
```

Comment:
Why do you need to reset? Should this maybe come with a big warning that you should only do this if you don't user this ledger for anything else?

Reply:
You need the Ledger device to use the same seed phrase that your principal ID has that was created with quill. As far as I understand your Ledger can only have one private key. Great point about the warning though!